staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -231,6 +238,37 @@ func (t *tracker) List(gvr schema.GroupVersionResource, gvk schema.GroupVersionK
 		return list, nil
 	}
 
+	// We need to return resourceVersion for the List
+	// It should be the highest resourceVersion in etcd for the resource
```

Comment:
this is only ok because logically different resources *could* have different, disjoint sets of RVs